### PR TITLE
Closes #745: Polish display for News content type nodes

### LINKED ISSUE #745: Polish display of news content type nodes
When we first built the news content type there were some outstanding issues with the design/display that we meant to clean up later:
-Width of the main content column is quite narrow on large screens
-Image is set to card image size and should probably be revisited
-Image may need cropping funcitonality?
-Subtitle is too small (this may be addressed in another ticket if the lead body text size is increased)

image

### PR DESCRIPTION
**Pull request by smayden**
Closes #745 

There were several elements we said we'd come back to later to polish how news stories appear for the news content type.

Before:
image

-Lead body copy not applying to subheading
-Column width appears too narrow on large displays
-Image not full width
-Caption same size as body text

## Description
Changes made by this PR:
-Set column width to `col-lg-10 offset-lg-1`
-Made headline `text-midnight` to match how we have it in other news displays
-Fixed bug where subhead not displaying as lead body copy size (would still love for this to be larger in bootstrap)
-Set image size to large
-Set caption to `small`

Column width is now similar to QS1 and gives about 85 characters max per line. We could make more readable by increasing both text size and line height in bootstrap. 

After:
image

Bonus: Updated order of fields on editing form to match same order as they display (since it was bugging me when entering demo content)

## Related Issue
#745 

## How Has This Been Tested?
Tested locally and in Probo

To test:
Create/view news stories at different display sizes

Probo: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-816.probo.build/news/fruit-flies-and-cellular-demise-zeroing-causes-als

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by joeparsons — APPROVED** [review]
Changes seem fine to me from a code perspective.

File: modules/custom/az_news/config/install/core.entity_view_display.node.az_news.default.yml [hunk 1]
```diff
@@ -192,13 +192,14 @@ third_party_settings:
         effect: none
         speed: fast
         id: ''
-        classes: 'border-bottom pb-3'
+        classes: 'border-bottom pb-2 small'
```
Comment (danahertzberg) [inline modules/custom/az_news/config/install/core.entity_view_display.node.az_news.default.yml:195]:
```suggestion
        classes: 'border-bottom mb-5 small'
```

**Review by danahertzberg — APPROVED** [review]

File: modules/custom/az_news/config/install/core.entity_form_display.node.az_news.default.yml [hunk 2]
```diff
@@ -85,15 +85,15 @@ content:
...
+    weight: 2
     settings:
       size: 60
       placeholder: ''
     third_party_settings: {  }
     type: string_textfield
     region: content
   field_az_caption:
-    weight: 4
+    weight: 5
```
Comment (danahertzberg) [inline modules/custom/az_news/config/install/core.entity_form_display.node.az_news.default.yml:96]:
```suggestion
    weight: 5
```
Testing force commit

**Review by joeparsons — APPROVED** [review]
(Re) approved.

**Pull request merged**